PythonTool/PlotContour.py

```diff
@@ -0,0 +1,31 @@
+from visit import *
+
+
+def PlotContour(File, ObjectSequence):
+    # Contour plot attributes.
+
+    # Change orientation of legend
+    LegendAttributes = GetAnnotationObject(ObjectSequence["Contour"])
+    LegendAttributes.orientation = "VerticalLeft"
+    LegendAttributes.fontHeight = 0.03
+
+    Attribute = ContourAttributes()
+
+    Attribute.legendFlag = 1
+
+    if File is not None:
```

Comment:
Lines 16-23 here are very much the same as in `PlotMesh.py`. Wondering if they could be factored out into a function which accepts "Contour" or "Mesh" as an argument.